src/minuteman_route_mgr.erl

```diff
@@ -73,9 +79,9 @@ start_link() ->
     {stop, Reason :: term()} | ignore).
 init([]) ->
     {ok, Pid} = gen_netlink_client:start_link(?NETLINK_ROUTE),
-    Routes = get_routes(Pid),
-    ensure_fib_rule(Pid),
-    {ok, #state{netlink = Pid, routes = Routes}}.
+    Iface = get_minuteman_iface(Pid),
```

Comment:
can't use the string in netlink apis, so we need to dynamically find the interface id based on the name.